tests/e2e/support/api/http.ts

```diff
@@ -9,25 +9,34 @@ export const request = async ({
   path,
   body,
   user,
-  formatJson = true
+  formatJson = true,
+  header = null
 }: {
   method: 'POST' | 'DELETE' | 'PUT' | 'GET' | 'MKCOL' | 'PROPFIND' | 'PATCH'
   path: string
   body?: BodyInit
   user?: User
   formatJson?: boolean
+  header?: object
 }): Promise<Response> => {
   const format = config.ocis || !formatJson ? '' : 'format=json'
 
+  let finalHeader = {}
+  const basicHeader = {
+    'OCS-APIREQUEST': true as any,
+    ...(user && {
+      Authorization: 'Basic ' + Buffer.from(user.id + ':' + user.password).toString('base64')
+    })
+  }
+  if (header) {
+    finalHeader = Object.assign(basicHeader, header)
+  }
+  const fullHeader = finalHeader ? basicHeader : finalHeader
+
```

Comment:
```suggestion
   let basicHeader = {
    'OCS-APIREQUEST': true as any,
    ...(user && {
      Authorization: 'Basic ' + Buffer.from(user.id + ':' + user.password).toString('base64')
    })
  }
  if (header) {
    basicHeader = Object.assign(basicHeader, header)
  }

```
can you check if it works like this?